proto/qdrant/cloud/cluster/backup/v1/backup.proto

```diff
@@ -60,6 +61,15 @@ service BackupService {
     // gRPC Gateway REST call
     option (google.api.http) = {post: "/api/cluster/backup/v1/accounts/{account_id}/backups/{backup_id}/restore"};
   }
+  // Create a new cluster from an existing backup
+  rpc CreateCluster(CreateClusterRequest) returns (CreateClusterResponse) {
```

Comment:
Why do we decide to add this method to the BackupService instead of the ClusterService (like `CreateClusterFromBackup`? The responsibility to create clusters belongs to the ClusterService and I see the backup as a reference, a different type of parameters used to create the cluster.

Reply:
> Why do we decide to add this method to the BackupService instead of the ClusterService (like `CreateClusterFromBackup`? The responsibility to create clusters belongs to the ClusterService and I see the backup as a reference, a different type of parameters used to create the cluster.

I see your point. I am not particularly biased here. Please let me know if I should take that route instead.

Reply:
> I agree with Toni, sorry for not spotting this earlier.

Addressed, ptal @Robert-Stam @areina